src/client/terminals/activation.ts

```diff
@@ -50,10 +50,15 @@ export class TerminalAutoActivation implements ITerminalAutoActivation {
         if ('hideFromUser' in terminal.creationOptions && terminal.creationOptions.hideFromUser) {
             return;
         }
-        // If we have just one workspace, then pass that as the resource.
-        // Until upstream VSC issue is resolved https://github.com/Microsoft/vscode/issues/63052.
+
+        const cwd =
+            'cwd' in terminal.creationOptions
+                ? terminal.creationOptions.cwd
+                : this.activeResourceService.getActiveResource();
+        const resource = typeof cwd === 'string' ? Uri.file(cwd) : cwd;
+        
         await this.activator.activateEnvironmentInTerminal(terminal, {
-            resource: this.activeResourceService.getActiveResource(),
+            resource: resource,
```

Comment:
```suggestion
            resource,
```
That is what it means by property short hand.

Reply:
If you install prettier extension, and set that as the formatter for TS. it should automatically do this for you.

Reply:
Yes, I'll fix this in a couple of hours from now! Thank you